src/robot_translation_node.py

```diff
@@ -42,10 +47,14 @@ def __init__(self):
         """ Initialize anything that needs initialization """
         # parse config file to find out which robot to send to
         try:
-            with open ("robot_translation_config.json") as json_file:
+            __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(sys.argv[0])))
```

Comment:
Why does getting and using the absolute path work better? I'm not against it; just curious, if you don't mind explaining your reasoning behind this change.